### PR TITLE
MYRIAD-196 - Allow remote distribution and configuration of JVM for executors

### PR DESCRIPTION
**Pull request by mohitsoni**
Made following configurable:
* JVM distribution URL
* JVM path relative to sandbox to assist in setting JAVA_HOME
* JAVA_LIBRARY_PATH

### CODE REVIEW

File: myriad-scheduler/src/main/resources/myriad-config-default.yml [hunk 1]
```diff
@@ -56,6 +56,9 @@ executor:
   #The following should be used for a remotely distributed URI, hdfs assumed but other URI types valid.
   #nodeManagerUri: hdfs://namenode:port/dist/hadoop-2.7.0.tar.gz
   #path: hdfs://namenode:port/dist/myriad-executor-runnable-0.1.0.jar
+  #jvmUri: https://downloads.mycompany.com/java/jre-7u76-linux-x64.tar.gz
+  #jvmPath: jre1.7.0_76
+  #javaLibraryPath: /opt/mycompany/lib
```
Comment (DarinJ) [inline myriad-scheduler/src/main/resources/myriad-config-default.yml]: We've already got a place to set JAVA_HOME by placing it in yarnEnvironment I've used it on my system alot so am comfortable saying it works.  Is there a good reason to add code for this environment variable?

I like the idea of allowing the option to d/l Java though!

Comment (mohitsoni) [inline myriad-scheduler/src/main/resources/myriad-config-default.yml]: That's a good point. I have updated the PR to re-using the existing mechanism for setting the env vars.

Comment (DarinJ) [inline myriad-scheduler/src/main/resources/myriad-config-default.yml]: Other than the fact you left javaPath and javaLibraryPath in the yaml file, looks good.  Should probably add a commented example in the yarnEnvironment map.

**Comment by DarinJ** [comment]
ship it!

**Pull request closed without merging**